app/serializers/alchemy/json_api/page_serializer.rb

```diff
@@ -17,16 +17,16 @@ class PageSerializer
       )
 
       belongs_to :language, record_type: :language, serializer: ::Alchemy::JsonApi::LanguageSerializer
-
       has_many :elements, record_type: :element, serializer: ::Alchemy::JsonApi::ElementSerializer
-      has_many :fixed_elements, record_type: :element, serializer: ::Alchemy::JsonApi::ElementSerializer
-
-      has_many :all_elements, record_type: :element, serializer: ::Alchemy::JsonApi::ElementSerializer do |page|
-        page.all_elements.select { |e| e.public? && !e.trashed? }
-      end
 
       with_options if: ->(_, params) { params[:admin] == true } do
-        attribute :tag_list
+        attribute :tag_list do |page|
+          Gutentag::Tagging.where(
```

Comment:
This is an `n+1` query when requesting multiple pages. It's probably a good idea to move that into a relation on the `Page` model

Reply:
I know :)

Since we do not support this flag from the controllers yet we can also remove this for the time being. The admin API is not even on the horizon.